packages/linter/src/rules/SxgDumpSignedExchangeVerify.ts

```diff
@@ -34,31 +72,65 @@ export class SxgDumpSignedExchangeVerify extends Rule {
           isValid: stdout.Valid,
           version: stdout.Version,
           uri: stdout.RequestURI,
-          status: stdout.ResponseStatus
+          status: stdout.ResponseStatus,
+          signatures: stdout.Signatures
         };
       });
     } catch (e) {
       return this.warn(
-        `not testing: couldn't execute '${CMD}' (not installed? not in PATH?)`
+        `not testing: couldn't execute [${CMD}] (not installed? not in PATH?)`
       );
     }
-    const debug = `${fetchToCurl(url, opt, false)} | ${CMD} ${ARGS.join(" ")}`;
+    const expected = {
+      isValid: true,
+      version: EXPECTED_VERSION,
+      uri: url,
+      status: 200
+    };
+    const diff = compare(expected, sxg);
+    if (Object.keys(diff).length !== 0) {
+      return this.fail(
+        `[${url}] is not valid [${JSON.stringify(diff)}] [debug: ${DEBUG}]`
+      );
+    }
+    const certUrl = sxg.signatures[0]["Params"]["cert-url"] as string;
+    if (!certUrl) {
+      return this.fail(`Can't find valid [cert-url] [${JSON.stringify(sxg)}]`);
+    }
     if (
-      !sxg.isValid ||
-      (sxg.uri !== url && sxg.version !== "1b3") ||
-      sxg.status !== 200
+      !(await urlHasContentType(
+        certUrl,
+        headers,
+        "application/cert-chain+cbor"
+      ))
     ) {
       return this.fail(
-        `[${url}] is not valid SXG [${JSON.stringify(sxg)}] [debug: ${debug}]`
+        `cert-url [${certUrl}] is not found or has wrong content type`
+      );
+    }
+    const validityUrl = sxg.signatures[0]["Params"]["validity-url"] as string;
+    if (!validityUrl) {
+      return this.fail(`Can't find valid [cert-url] [${JSON.stringify(sxg)}]`);
+    }
+    if (!(await urlHasContentType(validityUrl, headers, "application/cbor"))) {
+      return this.fail(
+        `validity-url [${validityUrl}] is not found or has wrong content type`
+      );
+    }
+    const expires = sxg.signatures[0]["Params"]["expires"] as number;
+    if (7 * 24 * 60 * 60 + Date.now() / 1000 < expires) {
```

Comment:
magic number, please replace with a constant

Reply:
Now `7 * SECONDS_IN_A_DAY`. Or you want the whole thing a constant?